src/Index.js

```diff
@@ -0,0 +1,321 @@
+const inquirer = require("inquirer");
+const fs = require("fs");
+const path = require("path");
+
+const Engineer = require("./lib/Engineer");
+const Intern = require("./lib/Intern");
+const Manager = require("./lib/Manager");
+
+//  Starter questions
+const teamRole = [
+  {
+    type: "list",
+    name: "role",
+    message: "Which employee role do you want to add?",
+    choices: ["Manager", "Engineer", "Intern"],
+  },
+];
+
+// Questions to be rendered depending on the role chosen
+const managerQuestions = [
```

Comment:
Add validation onto the questions to ensure the user has inputted something and/or it fits the requirements. 

For the name question, this could look something like: 
``validate: (name) =>  name ``

This means that name must be true i.e. something has been entered, before moving onto the next question. For more information on validation, see [here](https://www.npmjs.com/package/inquirer).